src/components/nav.jsx

```diff
@@ -6,7 +6,7 @@ import LogoIcon from 'images/svg/logo-icon.jsx';
 
 const Nav = () => (
   <nav className="nav">
-    <DynamicLink to="/" className="nav__logo">
+    <DynamicLink to="https://hackbeanpot.com/" className="nav__logo">
```

Comment:
I think it might be better to have the link as "/" since we're still deciding on hosting it at the domain vs. the subdomain